keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,210 @@
+/* Copyright 2018 GoTakigawa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #include QMK_KEYBOARD_H
+
+ extern keymap_config_t keymap_config;
+
+ // Each layer gets a name for readability, which is then used in the keymap matrix below.
+ // The underscores don't mean anything - you can have a layer called STUFF or any other name.
+ // Layer names don't all need to be of the same length, obviously, and you can also skip them
+ // entirely and just use numbers.
+ #define _QWERTY 0
+ #define _COLEMAK 1
+ #define _DVORAK 2
+ #define _LOWER 3
+ #define _RAISE 4
+ #define _EUCALYN 5
```

Comment:
If this is a "base" layer, like QWERTY, Colemak and Dvorak, you absolutely wan this below Lower and Raise, otherwise, you won't be able to access those layers if you switch to this layer.

https://docs.qmk.fm/#/keymap?id=keymap-layer-status

Also, you may want to consider using an enum here, so that these are easier to modify.

Eg: 
```c
enum layers {
  _QWERTY,
  _COLEMAK,
  _DVORAK,
  _EUCALYN,
  _LOWER,
  _RAISE,
  _ADJUST,
};
```